helm/kubernetes-coredns-chart/templates/configmap.yaml

```diff
@@ -30,6 +32,13 @@ data:
         proxy {{ . }}
         {{- end }}
         {{- end }}
+
+        {{- if .Values.configmap.custom }}
+        {{- range (.Values.configmap.custom | trimAll "\n " |  split "\n") }}
+        {{ . }}
+        {{- end }}
+        {{- end }}
```

Comment:
We need to render line by line or the line breaks in the text are escaped (again )and fails the conversion to json